server/src/test/proto/spine/test/delivery/delivery_events.proto

```diff
@@ -46,3 +46,17 @@ message DTaskAssigned {
 
     spine.core.UserId assignee = 2;
 }
+
+message PositiveNumberEmitted {
+
+    string id = 1;
+
+    int32 value = 2 [(min).value = "0"];
```

Comment:
Please add `exclusive = true` for this field and for `NegativeNumberEmitted. value`.

Reply:
Why cannot it be just "1" then?